examples/dask/.saturn/saturn.json

```
@@ -1,9 +1,9 @@
 {
-  "name": "dask",
-  "image_uri": "saturncloud/saturn:2021.11.10",
-  "description": "Use distributed computing with Dask",
-  "environment_variables": {
-    "SATURN__JUPYTER_SETUP_DASK_WORKSPACE": "true"
+  "name": "example-dask",
+  "image_uri": "public.ecr.aws/saturncloud/saturn:2022.01.06",
```

We should probably use the old image: 
```suggestion
  "image_uri": "saturncloud/saturn:2021.11.10",
```